.circleci/ci/it-tests.js

```diff
@@ -39,6 +39,7 @@ try {
         // Download the forms Add-On
         ci.sh(`curl -s "${process.env.FORMS_ADDON_URL}" -o forms-addon.far`);
         extras = '--install-file forms-addon.far';
+        extras += ` --bundle com.adobe.cq:core.wcm.components.all:${wcmVersion}:zip`;
```

Comment:
default jar gives version that is included within the quickstart, which currently is 2.17.4

latest version (2.18.0) is not yet available yet.

Reply:
This means if I merge this PR, core components artifact won't be usable in local SDK. Isn't that wrong ?

Reply:
this is only used for testing, right?
If a customer wants to use latest forms core component artifact, they'll need to update their wcm components version because we depend on it?
What is the expected behaviour?